rskj-core/build.gradle

```diff
@@ -189,18 +243,31 @@ task generatePom(dependsOn: jar) {
 }
 
 task fatJar(type: Jar, dependsOn: jar) {
+    archiveClassifier.set('all')
+    duplicatesStrategy = DuplicatesStrategy.EXCLUDE
+
     manifest {
         attributes 'Main-Class': "$mainClassName"
     }
-    exclude "META-INF/*.SF", "META-INF/*.DSA", "META-INF/*.RSA"
-    classifier = 'all'
+
+    exclude 'META-INF/*.SF', 'META-INF/*.DSA', 'META-INF/*.RSA'
+
+    exclude 'META-INF/NOTICE'
+    exclude 'META-INF/NOTICE.md'
+    exclude 'META-INF/NOTICE.txt'
+    exclude 'META-INF/LICENSE'
+    exclude 'META-INF/LICENSE.md'
+    exclude 'META-INF/LICENSE.txt'
+
+    exclude 'module-info.class'
```

Comment:
Are we removing this one for the same reasons as the previous chunk?

Reply:
Mentioned about this file in the comment above